doc/usage.rst

```diff
@@ -138,6 +138,39 @@ Alternative hashing algorithms supported by :mod:`hashlib` can be used if necess
     print(pooch.file_hash("data/c137.csv", alg="sha512"))
 
 
+Bypassing the hash check
+------------------------
+
+Sometimes we might not know the hash of the file or it could change on the server
+periodically. In these cases, we need a way of bypassing the hash check.
+One way of doing that is with Python's ``unittest.mock`` module, and setting the
+hash value to ``ANY`` (a special object which passes all equality tests).
```

Comment:
```suggestion
Sometimes we might not know the hash of the file or it could change on the server
periodically. In these cases, we need a way of bypassing the hash check.
One way of doing that is with Python's ``unittest.mock`` module. It defines the object
``unittest.mock.ANY`` which passes all equality tests made against it. To bypass the 
check, we can set the hash value to ``unittest.mock.ANY`` when specifying the 
``registry`` argument for :func:`pooch.create`.
```